### DISCUSSION TITLE
$post and $get helpers to wrap around $fetch

### DISCUSSION THREAD

**Discussion by KevinBatdorf** [alpine-collective/alpine-magic-helpers#104] [Ideas] [answered]
To do a post request you do: 

```js
$fetch({
  method: 'post',
  url: '/user/12345',
  data: {
    firstName: 'Fred',
    lastName: 'Flintstone'
  }
})
```
This is fine, but better would be:
```js
$post(url, options)
```
`$fetch` defaults to "get" but if we add `$post` it makes sense to have `$get` too

**Comment by SimoTod** [comment, marked as answer]
It makes sense and it's probably just a few bytes. I wonder if they should just be `$post(url, data)` and `$get(url)` to cover the simple cases. 
For example
```js
$post(
  '/user/12345'​,​
  {​​firstName​: ​'Fred'​, lastName​: ​'Flintstone'​}
)
```
If someone needs to pass custom options, they can use the full featured `$fetch`.
